src/details/ArborX_DetailsBatchedQueries.hpp

```diff
@@ -91,17 +91,18 @@ struct BatchedQueries
   static auto applyPermutation(Kokkos::View<size_t const *, DeviceType> permute,
                                Predicates const &v)
       -> Kokkos::View<
-          std::decay_t<
+          typename std::decay<
               decltype(Traits::Access<Predicates, Traits::PredicatesTag>::get(
-                  std::declval<Predicates const &>(), std::declval<int>()))> *,
+                  std::declval<Predicates const &>(),
+                  std::declval<int>()))>::type *,
```

Comment:
`clang format`?

Reply:
https://cloud.cees.ornl.gov/jenkins-ci/blue/organizations/jenkins/ArborX/detail/PR-132/1/pipeline

Reply:
I was more curious why the format changed - not that it was incorrectly formatted